src/to_tokens.rs

```diff
@@ -126,8 +127,58 @@ macro_rules! primitive {
 }
 
 primitive! {
-    i8 i16 i32 i64 isize
     u8 u16 u32 u64 usize
+}
+
+macro_rules! signed_primitive {
+    ($(($t:ident => $signed:ident))*) => ($(
```

Comment:
The $signed part is actually the unsigned version, and it's not used, could be removed.